components/button/stories/button.stories.js

```diff
@@ -95,35 +117,19 @@ export default {
 			},
 			options: ["white", "black"],
 			control: "select",
-			if: { arg: "isPendingStory", truthy: false },
-		},
-		showIconOnlyButton: {
-			table: {
-				disable: true,
-			},
-		},
-		layout: {
-			name: "Layout",
-			description: "How the buttons align in the preview (Storybook only).",
-			type: { name: "string" },
-			table: {
-				type: { summary: "string" },
-				category: "Advanced"
-			},
-			options: ["stacked","inline"],
-			control: "radio"
 		},
 	},
 	args: {
 		rootClass: "spectrum-Button",
 		size: "m",
 		label: "Edit",
-		variant: "accent",
 		treatment: "fill",
+		variant: "accent",
 		isDisabled: false,
 		isPending: false,
-		showIconOnlyButton: true,
-		layout: "inline",
+		isActive: false,
+		isFocused: false,
+		isHovered: false,
```

Comment:
Could these default args for the states also be added to the Action button stories.js file? That way the controls won't show up as the "Set boolean" button by default.